shaders/pbr-frag.glsl

```diff
@@ -289,8 +286,8 @@ void main()
     color += emissive;
   #endif
 
-  #ifdef USE_MATHS
```

Comment:
If you remove `USE_MATHS` here, please also remove it from the corresponding JavaScript.